From: Jubin John <jubin.john@intel.com>
Subject: staging/rdma/hfi1: Remove else after break
Git-commit: edddfca00eecd0949a9adccf8dd490478f641cbc
Patch-mainline: v4.6
References: bsc#973818, fate#319242
Acked-by: Jiri Bohac <jbohac@suse.cz>

Remove else after break to fix checkpatch warning:
WARNING: else is not generally useful after a break or return

Reviewed-by: Dennis Dalessandro <dennis.dalessandro@intel.com>
Reviewed-by: Ira Weiny <ira.weiny@intel.com>
Reviewed-by: Mike Marciniszyn <mike.marciniszyn@intel.com>
Signed-off-by: Jubin John <jubin.john@intel.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
---
 drivers/staging/rdma/hfi1/qp.c |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

--- a/drivers/staging/rdma/hfi1/qp.c
+++ b/drivers/staging/rdma/hfi1/qp.c
@@ -329,11 +329,9 @@ __be32 hfi1_compute_aeth(struct rvt_qp *
 			if (credit_table[x] > credits) {
 				max = x;
 			} else {
-				if (min == x) {
+				if (min == x)
 					break;
-				} else {
-					min = x;
-				}
+				min = x;
 			}
 		}
 		aeth |= x << HFI1_AETH_CREDIT_SHIFT;
